common/src/types/wallet.rs

```diff
@@ -321,7 +321,7 @@ pub mod mocks {
             fees: vec![],
             key_chain: KeyChain {
                 public_keys: PublicKeyChain {
-                    pk_root: PublicSigningKey::from(&BigUint::from(0u8)),
```

Comment:
this was done in a pr I just made, you should rebase on `joey/circuits-refactor`